src/verifai/scenic_server.py

```diff
@@ -23,25 +30,29 @@ def __init__(self, sampling_data, monitor, options={}):
         else:
             self.rejectionFeedback = extSampler.rejectionFeedback
         self.monitor = monitor
+        if isinstance(self.monitor, multi_objective_monitor):
+            self.sampler.scenario.externalSampler.sampler.domainSampler.split_sampler.samplers[0].set_graph(self.monitor.graph)
```

Comment:
This assumes the Scenic program is using an external sampler, and one of a particular form. Need a more generic solution to this. I think when we discussed this earlier I suggested passing the graph to the `ScenicSampler` and having the samplers pass this information down recursively (ideally when the samplers are being initialized, rather than calling something like `set_graph` after the fact).